src/main/kotlin/it/pagopa/wallet/eventdispatcher/configuration/WalletCDCClientConfiguration.kt

```diff
@@ -23,8 +23,13 @@ class WalletCDCClientConfiguration {
                 ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG to StringSerializer::class.java,
                 ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG to JsonSerializer::class.java,
                 ErrorHandlingDeserializer.VALUE_DESERIALIZER_CLASS to
-                    ErrorHandlingDeserializer::class.java
+                    ErrorHandlingDeserializer::class.java,
+                "sasl.jaas.config" to
+                    "org.apache.kafka.common.security.plain.PlainLoginModule required username=\"\$ConnectionString\" password=\"${walletCDCConfiguration.connectionString}\";",
+                "sasl.mechanism" to "PLAIN",
+                "security.protocol" to "SASL_SSL"
```

Comment:
Please consider using CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, SaslConfigs.SASL_MECHANISM and SaslConfigs.SASL_JAAS_CONFIG constants instead of plain string

Reply:
done here 89ba77e02d3bbda40291862e32f904ce967cee8b